swagger/producer-static/narrative.yaml

```diff
@@ -1117,6 +1120,18 @@ components:
         SNOMED_CODES_NATIONAL_EARLY_WARNING_SCORE_2:
           summary: National Early Warning Code 2
           value: http://snomed.info/sct|1363501000000100
+        SNOMED_CODES_RESPECT_FORM:
+          summary: ReSPECT form
+          value: http://snomed.info/sct|1382601000000107
+        SNOMED_CODES_NEWS2_CHART:
+          summary: Royal College of Physicians NEWS2
+          value: http://snomed.info/sct|1363501000000100
```

Comment:
SNOMED_CODES_NEWS2_CHART is a duplicate to SNOMED_CODES_NATIONAL_EARLY_WARNING_SCORE_2